game/scripts/npc/items/item_helm_of_the_dominator.txt

```diff
@@ -62,12 +62,12 @@
       "01"
 			{
 				"var_type"				"FIELD_INTEGER"
-				"bonus_stats"			"2 25 40 55 70"
+				"bonus_stats"			"2 20 30 45 60"
 			}
 			"02"
 			{
 				"var_type"				"FIELD_INTEGER"
-        "attack_speed_aura"   "20 35 60 80 100"
+        "attack_speed_aura"   "20 30 45 60 80"
```

Comment:
even with both changes hotd is by far the best attack speed item. Can make the mid levels scale a bit better, they are currently low because they scaled so strangely previously (why did it jump from 35 to 60?)

Reply:
This item is a monster on agility heroes.

Reply:
i like this change